internal/env/env.go

```diff
@@ -0,0 +1,37 @@
+package env
+
+const (
+	DevboxCache            = "DEVBOX_CACHE"
+	DevboxCLICloudShell    = "DEVBOX_CLI_CLOUD_SHELL"
+	DevboxDebug            = "DEVBOX_DEBUG"
+	DevboxFeaturePrefix    = "DEVBOX_FEATURE_"
+	DevboxGateway          = "DEVBOX_GATEWAY"
+	DevboxNotUpgradeConfig = "DEVBOX_DONT_UPGRADE_CONFIG"
```

Comment:
nit: can we call this `DevboxDoNotUpgradeConfig` ?

Reply:
maybe send a small followup PR?

Reply:
Done. PTAL